### DISCUSSION TITLE
How to upgrade when Part-DB is in a Docker Container

### DISCUSSION THREAD

**Discussion by influx3k** [Part-DB/Part-DB-server#455] [General]
Hi, what is the official upgrade procedure, when upgrading Part-DB in a Docker container?  I've tried to upgrade two times now and both times resulted in catastrophic failure and the app being down.  Is there a procedure that can be followed so that a successful upgrade can happen with confidence?

**Comment by jbtronics** [comment]
See here: https://docs.part-db.de/installation/installation_docker.html#update-part-db

The `--user=www-data` option in the database migration (or any other command executed inside Part-DB) is important. Otherwise the permissions of the cache gets wrong.

**Reply by influx3k** [reply]
Ok, awesome.  Thanks for the info.
